StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/STPElementsSession.swift

```diff
@@ -11,6 +11,9 @@ import Foundation
 
 /// The response returned by v1/elements/sessions
 final class STPElementsSession: NSObject {
+    #if DEBUG && targetEnvironment(simulator)
+    public static let countryCodeOverride: String? = nil
```

Comment:
Do we need to scope this to DEBUG/simulator? I can see pros and cons, but if we do this it won't work in TestFlight builds. Maybe that's okay, but we should point that out in the description in the test playground.

Reply:
Hmmm -- I've changed this to DEBUG scope only, and updated the playground to note debug only